src/Generator.Bind/Specifications/GL2/GL/4.5/ARB_direct_state_access.xml

```diff
@@ -220,9 +220,6 @@
     <function name="VertexArrayAttribFormat">
       <param name="type"><type>VertexAttribType</type></param>
     </function>
-    <function name="VertexArrayAttribIFormat">
-      <param name="type"><type>VertexAttribIntegerType</type></param>
-    </function>
```

Comment:
Why did you have to remove this? What happens if you do not remove this?

Reply:
Without this there is no overload for `VertexArrayAttribIFormat` that accepts  `VertexAttribType` as the type.
Notice that this does not occur with the "L" version. It has an overload for `VertexAttribType` (backward compat)  and `VertexAttribDoubleType`.

Reply:
Shouldn't this add one for `VertexAttribIntegerType` so you don't have to add it to the other file?

Reply:
Yeah it seems to behave like that - even if I don't touch `signatures.xml` this will add an overload for `VertexAttribIntegerType`. However it will also leave the overrides added in `overrides.xml` to avoid breaking changes ignored.
The way I solved that here is by removing it and instead making modifications in `signatures.xml`. That way the overrides are not ignored and the functions have an overload for `VertexAttribIntegerType`/`VertexAttribDoubleType`.

Reply:
I'm just weary of changing `ARB_direct_state_access.xml` file, but I guess it's fine.